daemon/pod.go

```diff
@@ -42,11 +43,201 @@ type Pod struct {
 	sync.RWMutex
 }
 
-func NewPod(rawSpec []byte, id string, data interface{}) (*Pod, error) {
+// TODO: remove convertToRunvPodSpec after pod.UserPod is deleted from runv
+func convertToRunvPodSpec(podSpec *apitypes.UserPod) (*pod.UserPod, error) {
```

Comment:
No, `apitypes.UserPod` is for all hyperd APIs and clients. Users can see it directly:

- For gRPC api, users must specify `apitypes.UserPod` while creating Pods
- For original restful api, the `pod.json` is unmarshaled to `apitypes.UserPod`

After this PR, hyperd APIs will no longer depend on `UserPod` in runv. 

> I don't think apitypes.UserPod can replace pod.UserPod. 

Why can't?